src/PrestaShopBundle/Form/Admin/Improve/International/Tax/TaxOptionsType.php

```diff
@@ -52,38 +53,89 @@ class TaxOptionsType extends AbstractType
      */
     private $taxRuleGroupChoiceProvider;
 
+    /**
+     * @var TranslatorInterface
+     */
+    private $translator;
+
     /**
      * TaxOptionsType constructor.
      *
      * @param bool $ecoTaxEnabled
      * @param FormChoiceProviderInterface $taxAddressTypeChoiceProvider
      * @param FormChoiceProviderInterface $taxRuleGroupChoiceProvider
+     * @param TranslatorInterface $translator
      */
     public function __construct(
         $ecoTaxEnabled,
         FormChoiceProviderInterface $taxAddressTypeChoiceProvider,
-        FormChoiceProviderInterface $taxRuleGroupChoiceProvider
+        FormChoiceProviderInterface $taxRuleGroupChoiceProvider,
+        TranslatorInterface $translator
     ) {
         $this->ecoTaxEnabled = $ecoTaxEnabled;
         $this->taxAddressTypeChoiceProvider = $taxAddressTypeChoiceProvider;
         $this->taxRuleGroupChoiceProvider = $taxRuleGroupChoiceProvider;
+        $this->translator = $translator;
     }
 
     /**
      * {@inheritdoc}
      */
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
-        $builder->add('enable_tax', SwitchType::class)
-            ->add('display_tax_in_cart', SwitchType::class)
+        $builder->add('enable_tax', SwitchType::class, [
```

Comment:
```suggestion
        $builder->add(
            'enable_tax', 
            SwitchType::class, [
```

If you want a better indentation (look at line 97, it's really weird)